src/mscorlib/model.xml

```diff
@@ -8608,6 +8608,14 @@
       <Member Name="GetHashCode" />
       <Member Name="ToString" />
       <Member MemberType="Property" Name="Item1" />
+      <Member Status="ImplRoot" Name="System.Collections.IStructuralComparable.CompareTo(System.Object,System.Collections.IComparer)" />
```

Comment:
The corelib pruning tool automatically keeps interface implementations around. It is not necessary to mention them explicitly.